### ISSUE TITLE
security question : does use of the built-in google accounts register with my device when they're used?

### ISSUE THREAD

**Issue by jawz101** [yeriomin/YalpStore#287]
I could imagine this app collecting all of our devices on some account which would let you remotely wipe our devices.  Just wondering how that is not a part of the equation.

**Comment by yeriomin** [comment]
>does use of the built-in google accounts register with my device when they're used?

I'm not sure I understand correctly, but probably yes. Your real device info is used by default. App compatibility filtering depends on it.

>remotely wipe our devices

For that I would need your google account username and password and using the built-in account does not give me that. Using your real account with Yalp is a different story. Also, you have no reason to trust my app or my claims, so it is a useless question. The source is open, so you can audit the source or just delete the app.

Points to consider:

1. All the information collected from the device to log in to Google is freely available to **any other app on your device without any permission**. So any app you already have can do the same.
2. "Internet" permission is considered safe in Android, so it will not even make apps ask for that explicitly, so the information from 1. can be sent somewhere without your consent in background by any other app on your device.

Tl;dr the information collected from the device cannot be used for anything nefarious even if I wanted to.

**Comment by jawz101** [comment]
Thank you very much
